cmake/ci.cmake

```diff
@@ -928,6 +928,7 @@ add_custom_target(ci_icpc
         -DCMAKE_BUILD_TYPE=Debug -GNinja
         -DCMAKE_C_COMPILER=icc -DCMAKE_CXX_COMPILER=icpc
         -DJSON_BuildTests=ON -DJSON_FastTests=ON
+        -DCMAKE_CXX_FLAGS="-D_MSC_VER=0 -D_MSC_FULL_VER=0"
```

Comment:
```suggestion
        -DCMAKE_CXX_FLAGS="-D_MSC_VER=0;-D_MSC_FULL_VER=0"
```

Reply:
Thanks for the hint. But since this was a only a hack in the first place, I'd undo it and wait for a fix from upstream.

Reply:
Just a reminder to use semicolons when dealing with CMake lists in the future. :-)

Reply:
This was not the first one, and I fear it was also not the last one :-)